src/client/datascience/kernel-launcher/kernelEnvVarsService.ts

```diff
@@ -69,24 +69,25 @@ export class KernelEnvironmentVariablesService {
         this.envVarsService.mergeVariables(interpreterEnv, mergedVars); // interpreter vars win over proc.
         this.envVarsService.mergeVariables(kernelEnv, mergedVars); // kernels vars win over interpreter.
         this.envVarsService.mergeVariables(customEditVars, mergedVars); // custom vars win over all.
-        if (customEditVars[this.platformService.pathVariableName]) {
-            this.envVarsService.appendPath(mergedVars, customEditVars[this.platformService.pathVariableName]!);
+        const pathVariable = this.platformService.pathVariableName;
+        if (process.env[pathVariable]) {
```

Comment:
mergedVars is initialized from process.env and mergeVariables is now back to ignoring PATH and PYTHONPATH so these two ifs don't look needed. If process.env has a PATH or PYTHONPATH it's already in mregedVars and none of the merges above would overwrite them.